test/controllers/external_pledges_controller_test.rb

```diff
@@ -57,9 +58,9 @@ class ExternalPledgesControllerTest < ActionDispatch::IntegrationTest
 
     it 'should produce an audit trail' do
       assert_equal @pledge.history_tracks.count, 2
-      @changes = @pledge.history_tracks.last
-      assert_equal @changes.modified[:updated_by_id], @user.id
-      assert_equal @changes.modified[:source], 'Edited Pledge'
+      last_changes = @pledge.history_tracks.last
+      assert_equal last_changes.updated_by_id, @user.id
```

Comment:
I'm not sure what about removing the `gon` gem commit makes `updated_by_id` not be included in the list of modified fields, but we can get this from a different spot so I'm not worried about it.

Also renamed `@changes` to `last_changes` to cut down on an unnecessary instance variable.